### DISCUSSION TITLE
关于vip+节点的问题

### DISCUSSION THREAD

**Discussion by NorthShad0w** [natfrp/wiki#73] [使用问题] [answered]
我用vip节点能正常穿透rdp服务
用vip+里的节点就不行 提示内部错误
是拦截了流量么
要不标注一下？

**Comment by fakeboboliu** [comment, marked as answer]
请看站点公告：
![image](https://user-images.githubusercontent.com/7552030/172142306-97bc9de3-ba3a-45ef-bd00-fcec59fa01d7.png)

在创建隧道界面的确缺少一些提示，我们将考虑尝试在不破坏现有体验和操作的情况下提供一个提示。

[a comment by an automated account is not shown]
